testdata/.tranor/envs.json

```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"name": "dev",
+		"dnsSuffix": "dev.mycompany.com"
+	},
+	{
+		"name": "stg",
+		"dnsSuffix": "stg.mycompany.com"
+	},
+	{
+		"name": "prd",
+		"dnsSuffix": "mycompany.com"
+	}
+]
```

Comment:
This is a test file. The names aren't actually relevant. But I updated the code to use dev, qa, stage and production.